doc/nrf/ug_nrf9160.rst

```diff
@@ -7,9 +7,9 @@ Working with nRF9160 DK
    :local:
    :depth: 2
 
-The nRF9160 DK is a hardware development platform used to design and develop application firmware on the nRF9160 LTE Cat-M1 and Cat-NB1 :term:`System in Package (SiP)`.
+The nRF9160 development kit (DK) is a hardware development platform used to design and develop application firmware on the nRF9160 LTE Cat-M1 and Cat-NB1 :term:`System in Package (SiP)`.
```

Comment:
Should be 'nRF9160 DK' without spelling out DK since it is the product name. See https://projecttools.nordicsemi.no/confluence/pages/viewpage.action?spaceKey=MAR&title=Product+names

Reply:
Fixed.
Also fixed on line 10 of Getting Started.